build/roslib.js

```diff
@@ -1565,12 +1589,9 @@ function TFClient(options) {
 TFClient.prototype.processTFArray = function(tf) {
   var that = this;
   tf.transforms.forEach(function(transform) {
-    var frameID = transform.child_frame_id;
-    if (frameID[0] === '/') {
-      frameID = frameID.substring(1);
-    }
-    var info = that.frameInfos[frameID];
-    if (info !== undefined) {
+    var frameID = transform.child_frame_id.trimLeft('/');
+    var info = this.frameInfos[frameID];
```

Comment:
Good stuff! Note, you are changing the wrong file. This is just a concatenated build artifact; Search for 'frameID.substring' in the repo - there's the file actually defining the TFClient.